source/proximity_list/proximity_list.cpp

```diff
@@ -1,9 +1,158 @@
 #include "proximity_list.h"
+#include "type/geographical_coord.h"
+
+#include <cmath>
+#include <array>
+#include <exception>
+#include <flann/flann.hpp>
 
 namespace navitia {
 namespace proximitylist {
+using type::GeographicalCoord;
+
+static std::array<float, 3> project_coord(const type::GeographicalCoord& coord) {
+    std::array<float, 3> res;
+    float lat_grad = coord.lat() * GeographicalCoord::N_DEG_TO_RAD;
+    float lon_grad = coord.lon() * GeographicalCoord::N_DEG_TO_RAD;
+    res[0] = GeographicalCoord::EARTH_RADIUS_IN_METERS * cos(lat_grad) * sin(lon_grad);
```

Comment:
If `float` enough precision wise (compare to double)?

Reply:
i changed that to double...